rockpaperscissors

```diff
@@ -0,0 +1 @@
+Subproject commit 6b1be4b1ba4ea1d40237cc75a3865ee63f2ad067
```

Comment:
I assume this is because this git repository is distinct from the gem's git repository. That can be found here: https://github.com/dsberger/Rockpaperscissors